Skills/Perk/PregnancyShakyLegs.gd

```diff
@@ -0,0 +1,22 @@
+extends PerkBase
+
+func _init():
+	id = Perk.PregnancyShakyLegs
+	skillGroup = Skill.Pregnancy
+
+func getVisibleName():
+	return "Shaky legs"
+
+func getVisibleDescription():
+	# For now this works as - if you see a 20+ cm dick you go in heat, might make it chance based depending on the length
+	return "You are wild about big hard juicy cocks and the thoughts of getting filled and knocked up by them. Seeing one, your legs become all shaky and you are forced into heat."
```

Comment:
I.. I dunno. The implementation would be a hacky nightmare and it's kinda unrealistic (even more than usually x3). A simplier idea would be something like: Receive heat if you orgasm from someone with a big dick. Would need some code changes too but way less I think

But I suggest rethinking this one..

Reply:
Replaced with Submissive and breedable perk.